src/sentry/auth/access.py

```diff
@@ -955,7 +955,7 @@ def from_request_org_and_scopes(
             org_member=member,
         )
 
-        superuser_scopes = get_superuser_scopes(auth_state, member)
+        superuser_scopes = get_superuser_scopes(auth_state, request.user)
```

Comment:
I thought request.user was a User object, not an org member

Reply:
it is. i changed the typing to reflect this

Reply:
Oh I see, could you change the arg name to `user` then

Reply:
ah i see